rom-hacking/scripting/animation-commands/ac-mark-gyoe.md

```diff
@@ -0,0 +1,23 @@
+# (783) _AC_MARK_GYOE
+
+## Effect
+
+Displays an exclamation mark bubble \[!\] above the entity's head.
```

Comment:
This command takes an argument in multiple vanilla scripts, is this a stubbed feature?

Reply:
Seems to be stubbed https://github.com/TeamLumi/luminescent-team/pull/84#discussion_r1379046284